setup.cfg

```diff
@@ -89,12 +88,12 @@ filterwarnings =
     ignore:distutils Version classes are deprecated:DeprecationWarning
     ignore:Passing unrecognized arguments to super:DeprecationWarning
     ignore:.*With traitlets 4\.1, metadata should be set using the \.tag\(\) method:DeprecationWarning
-    ignore::DeprecationWarning:spectral_cube
     ignore::DeprecationWarning:glue
     ignore::DeprecationWarning:bqplot
     ignore::DeprecationWarning:bqplot_image_gl
     ignore::DeprecationWarning:bqscales
     ignore::DeprecationWarning:traittypes
+    ignore::DeprecationWarning:voila
```

Comment:
I see a new warning locally, so I need this line so the tests would run on my machine.